linksight/accounts/forms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
```

Comment:
I think the alias doesn't help with readability

Reply:
It's convention. I don't even want to use it.

Reply:
ZAD